js/framework/AnalyticsConstants.js

```diff
@@ -564,6 +573,20 @@ if (!OO.Analytics.EVENT_DATA)
 
   /**
    * @public
+   * @class Analytics.EVENT_DATA#VideoErrorData
+   * @classdesc Contains information about the error code and message of the video error.
+   * @property {number} errorCode The error code
+   * @property {string} errorMEssage The message that corresponds to the error code
+   */
+  EVENT_DATA.VideoErrorData = function(errorCode, errorMessage)
```

Comment:
Can you create another constant (at the end of the file) and start our error code enum? It can just start with one entry "unknown error" or "generic error"